modules/nodes-descheduler-configuring-profiles.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/nodes-descheduler.adoc
+
+[id="nodes-descheduler-configuring-profiles_{context}"]
+= Configuring descheduler profiles
+
+You can configure which profiles that the descheduler uses to evict pods.
+
+.Prerequisites
+* Cluster administrator privileges.
+
+.Procedure
+
+. Edit the `KubeDescheduler` object:
++
+[source,terminal]
+----
+$ oc edit kubedeschedulers.operator.openshift.io cluster -n openshift-kube-descheduler-operator
+----
+
+. Specify one or more profiles in the `spec.profiles` section.
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1beta1
+kind: KubeDescheduler
+metadata:
+  name: cluster
+  namespace: openshift-kube-descheduler-operator
+spec:
+  deschedulingIntervalSeconds: 3600
+  logLevel: Normal
+  managementState: Managed
+  operatorLogLevel: Normal
+  profiles:
+  - AffinityAndTaints       <1>
+  - TopologyAndDuplicates   <2>
+  - LifecycleAndUtilization <3>
+----
+<1> Enable the `AffinityAndTaints` profile, which evicts pods in an effort to evenly spread similar pods, or pods of the same topology domain, among nodes.
+<2> Enable the `TopologyAndDuplicates` profile, which evicts pods that violate basic pod and node affinity and node taints.
+<3> Enable the `LifecycleAndUtilization` profile, which evicts long-running pods and balances resource usage between nodes.
```

Comment:
@bergerhoffer definitions for 1 & 2 are reversed again here. Could you please help take a look ?

<1> Enable the `AffinityAndTaints` profile, which evicts pods that violate basic pod and node affinity and node taints.

<2> Enable the `TopologyAndDuplicates` profile, which evicts pods in an effort to evenly spread similar pods, or pods of the same topology domain, among nodes.